core-bundle/src/Image/Preview/ImaginePreviewProvider.php

```diff
@@ -100,21 +100,8 @@ private function getDimensionsFromImageSize(BoxInterface $imageSize, int $size):
 
     private function imagineSupportsFormat(string $format): bool
     {
-        // TODO: Use once Imagine 1.3.0 was released
-        // if ($this->imagine instanceof InfoProvider) {
-        //    return $this->imagine->getDriverInfo()->isFormatSupported($format);
-        // }
-
-        if ($this->imagine instanceof ImagickImagine) {
-            return \in_array(strtoupper($format), \Imagick::queryFormats(strtoupper($format)), true);
-        }
-
-        if ($this->imagine instanceof GmagickImagine) {
-            return \in_array(strtoupper($format), (new \Gmagick())->queryformats(strtoupper($format)), true);
-        }
-
-        if ($this->imagine instanceof GdImagine) {
-            return \function_exists('image'.$format);
+        if ($this->imagine instanceof InfoProvider) {
+            return $this->imagine->getDriverInfo()->isFormatSupported($format);
         }
```

Comment:
It looks like `getDriverInfo()->isFormatSupported()` does not handle all formats (e.g. PDF), so we can only use it as a fallback I think. 

@Toflar can you test if the current state of this PR would work for PDF files? If not we have to change it to something like this:

```suggestion
        if ($this->imagine instanceof ImagickImagine) {
            return \in_array(strtoupper($format), \Imagick::queryFormats(strtoupper($format)), true);
        }

        if ($this->imagine instanceof GmagickImagine) {
            return \in_array(strtoupper($format), (new \Gmagick())->queryformats(strtoupper($format)), true);
        }

        if ($this->imagine instanceof GdImagine) {
            return \function_exists('image'.$format);
        }

        if ($this->imagine instanceof InfoProvider) {
            return $this->imagine->getDriverInfo()->isFormatSupported($format);
        }
```

Reply:
I have no clue, I just implemented the `TODO`. But looking at the code, it looks like only https://github.com/php-imagine/Imagine/blob/develop/src/Image/Format.php#L14-L32 are supported. So this will never make any sense for the `ImaginePreviewProvider`. Might as well just drop the TODO.